scale-testing.mk

```
@@ -5,13 +5,22 @@
 AWS_REGION ?= us-east-2
 EKS_K8S_VERSION ?= 1.30
 
+#OPERATOR_NAMESPACE ?= vault-secrets-operator-system
+#IMAGE_TAG_BASE ?= hashicorp/vault-secrets-operator
+#K8S_VAULT_NAMESPACE ?= vault
```

```suggestion
#OPERATOR_NAMESPACE ?= vault-secrets-operator-system
#IMAGE_TAG_BASE ?= hashicorp/vault-secrets-operator
#K8S_VAULT_NAMESPACE ?= vault
```
nit: could remove them if no longer used?